SiaInterface/bin/Renter.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:http/http.dart' as http;
+import 'dart:io';
+
+class Renter {
+
+   /*
+   * nickname:     file that has been uploaded to the network.
+   * destination:  the path that the file will be downloaded to.
+   */
+
+  download(String nickname, String destination) {
+    var url = "http://localhost:9980/renter/download";
+
+    http.post(url, body: 'name=doodle&color=blue').then((response) {
+       print("Response status: ${response.statusCode}");
+       print("Response body: ${response.body}");
+     });
+
+  }
+  
+  downloadqueue() {
+
+    var url = "http://localhost:9980/renter/downloadqueue";
+
+    http.get(url).then((response){
+      print("Response status: ${response.statusCode}");
+          print("Response body: ${response.body}");
+    });
+
+  }
+  
+  files() {
```

Comment:
all these functions should have a return type- `void` for nothing and `YourCustomClass` or `bool` for ones that return meaningful responses (which should be almost if not all of them)